platform-api/src/main/java/com/flow/platform/api/util/PlatformUrlUtil.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2017 flow.ci
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.flow.platform.api.util;
+
+import javax.annotation.PostConstruct;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+/**
+ * @author yh@firim
+ */
+
+@Component(value = "platformUrlUtil")
+public class PlatformUrlUtil {
```

Comment:
Name to PlatformURL, remove @Component

class PlatformURL {
    public PlatformURL(String base) {
         xxxx
    }
}

In AppConfig

 @Value(value = "${platform.url}")
private String baseUrl;

@Bean
public PlatfromURL platformURL() {
   return new PlatformURL(baseURL)
}